### PR TITLE
add data locality defaults to storage class

### PR DESCRIPTION
**Pull request by janesser**
Adopt feature from https://github.com/longhorn/longhorn/issues/1045 for helm charts.

### CODE REVIEW

**Comment by janesser** [comment]
@innobead Hi, any chance this gets reviewed & merged "one of these days"? BR, Jan

File: charts/longhorn/templates/storageclass.yaml [hunk 1]
```diff
@@ -14,11 +14,12 @@ data:
         storageclass.kubernetes.io/is-default-class: {{ .Values.persistence.defaultClass | quote }}
     provisioner: driver.longhorn.io
     allowVolumeExpansion: true
-    reclaimPolicy: "{{ .Values.persistence.reclaimPolicy }}"
+    reclaimPolicy: {{ .Values.persistence.reclaimPolicy | quote }}
```
Comment (jenting) [inline charts/longhorn/templates/storageclass.yaml:17]: This one does not need to be quoted.

**Review by jenting — COMMENTED** [review]
Thank you @janesser for the contribution, sorry for the late reply, we missed this PR before.

Could you please file PR to [here](https://github.com/longhorn/longhorn/tree/master/chart)? Because this repo is mirrored from the https://github.com/longhorn/longhorn/tree/master/chart, thank you.

**Comment by janesser** [comment]
Proposing PR for longhorn repo.

@jenting Thanks for picking me up. May i suggest to turn this one read-only.

**Pull request closed without merging**